src/palette/internal/palettecreator.cpp

```diff
@@ -1235,45 +1189,28 @@ PalettePtr PaletteCreator::newLinesPalette(bool defaultPalette)
     volta->setEndings(il);
     sp->appendElement(volta, QT_TRANSLATE_NOOP("palette", "Seconda volta, open"));
 
-    auto ottava = makeElement<Ottava>(gpaletteScore);
-    ottava->setOttavaType(OttavaType::OTTAVA_8VA);
-    ottava->setLen(w);
-    ottava->styleChanged();
-    sp->appendElement(ottava, QT_TRANSLATE_NOOP("palette", "8va alta"));
-
-    ottava = makeElement<Ottava>(gpaletteScore);
-    ottava->setOttavaType(OttavaType::OTTAVA_8VB);
-    ottava->setLen(w);
-    ottava->setPlacement(PlacementV::BELOW);
-    ottava->styleChanged();
-    sp->appendElement(ottava, QT_TRANSLATE_NOOP("palette", "8va bassa"));
-
-    ottava = makeElement<Ottava>(gpaletteScore);
-    ottava->setOttavaType(OttavaType::OTTAVA_15MA);
-    ottava->setLen(w);
-    ottava->styleChanged();
-    sp->appendElement(ottava, QT_TRANSLATE_NOOP("palette", "15ma alta"));
-
-    if (!defaultPalette) {
-        ottava = makeElement<Ottava>(gpaletteScore);
-        ottava->setOttavaType(OttavaType::OTTAVA_15MB);
-        ottava->setLen(w);
-        ottava->setPlacement(PlacementV::BELOW);
-        ottava->styleChanged();
-        sp->appendElement(ottava, QT_TRANSLATE_NOOP("palette", "15ma bassa"));
-
-        ottava = makeElement<Ottava>(gpaletteScore);
-        ottava->setOttavaType(OttavaType::OTTAVA_22MA);
-        ottava->setLen(w);
-        ottava->styleChanged();
-        sp->appendElement(ottava, QT_TRANSLATE_NOOP("palette", "22ma alta"));
-
-        ottava = makeElement<Ottava>(gpaletteScore);
-        ottava->setOttavaType(OttavaType::OTTAVA_22MB);
-        ottava->setPlacement(PlacementV::BELOW);
+    static std::vector<OttavaType> ottavasDefault {
+        OttavaType::OTTAVA_8VA,
+        OttavaType::OTTAVA_8VB,
+        OttavaType::OTTAVA_15MA
+    };
+    static std::vector<OttavaType> ottavasMaster {
+        OttavaType::OTTAVA_8VA,
+        OttavaType::OTTAVA_8VB,
+        OttavaType::OTTAVA_15MA,
+        OttavaType::OTTAVA_15MB,
+        OttavaType::OTTAVA_22MA,
+        OttavaType::OTTAVA_22MB
+    };
+    for (OttavaType ottavaType : defaultPalette ? ottavasDefault : ottavasMaster) {
+        auto ottava = makeElement<Ottava>(gpaletteScore);
+        ottava->setOttavaType(ottavaType);
         ottava->setLen(w);
+        if (!ottava->placeAbove()) {
+            ottava->setPlacement(PlacementV::BELOW);
+        }
```

Comment:
Actually it looks like these lines will do nothing, because `!placeAbove()` seems to mean `!(placement() == PlacementV::Above)`, i.e. `placement() == PlacementV::Below` already.